Caramelo18/Visual-Kur
# Ignore unrecognized nodes

## What was added
1. Ignoring nodes

## How to Test
1. Open the app
2. Create a new node in the text editor with an unrecognized name (test, for instance)
3. Watch as it doesn't appear on the visual representation

## Notes for the reviewer
Most of the changes are indentation/beautifiers because it was hurting my eyes.

Only relevant one is the one [here](https://github.com/Caramelo18/Visual-Kur/blob/38c61882ba0667f97dd0f4fcd21242dd4901342d/src/containers/App.js#L124), which is the addition of the following if statement to ignore the empty nodes:

```javascript
if (Object.keys(layer).length === 0) {
    return null;
}
```

## Screenshot

Node "test" does not appear on the visual representation:
![screenshot from 2018-06-11 19-53-13](https://user-images.githubusercontent.com/12097873/41251555-2eecd654-6db2-11e8-90b8-6997d972ed79.png)
